src/components/forms/ManagementForm.jsx

```diff
@@ -233,6 +244,18 @@ export const ManagementForm = () => {
                 <Divider />
                 <Preferences />
                 <Divider />
+                <Item
+                    rules={[
+                        {
+                            required: true,
+                        },
+                    ]}
+                    label="Second language chosen"
+                    name="second_language"
+                >
+                    <Input placeholder="Eg: Hindi" />
+                </Item>
```

Comment:
This is OK.
But since we use this in both forms, can be made as a separate component.